ivy/func_wrapper.py

```diff
@@ -31,6 +31,7 @@
     "with_unsupported_dtypes",
     "handle_nans",
     "handle_mixed_function",
+    "handle_device_shifting_for_arrays",
```

Comment:
So, the above list decides the order of executing the decorator for each function? Does that mean I only need to update the decorator position in the list and keep the rest of the code unchanged?

Reply:
So this list decides the sequence by which the function would be wrapped when a backend is set. When using ivy without setting a backend, it uses the sequence specified above each function

Reply:
Since the decorator location is to be moved for all ivy functions, I will just confirm what I understood - `handle_device_shifting_for_arrays` should be below `infer_device` in the list and the decorators placed on ivy functions should be in the same order as in the `FN_DECORATORS` list, right?